pypeit/tests/test_pypeitfile.py

```diff
@@ -0,0 +1,67 @@
+""" Tests on PypeItFile """
+
+import os
+import pytest
+
+from astropy.table import Table
+
+from pypeit.inputfiles import PypeItFile
+from pypeit.tests.tstutils import data_path
+
+# Bits needed to generate a PypeIt file
+confdict = {'rdx': {'spectrograph': 'keck_hires'}}
+
+data = Table()
+data['filename'] = ['b1.fits.gz', 'b27.fits.gz']
+data['frametype'] = ['arc', 'science']
+data['exptime'] = [1., 10.]
+
+file_paths = [data_path('')]
+
+setup_dict = {'Setup A': ' '}
```

Comment:
I'm (overly?) picky about object scope.  All of the above are quick to generate, so I'd prefer if they're only declared within the test function where they're needed.

Reply:
Fair enough.  I pushed them into a simple method